pkcs11js/pkcs11js.d.ts

```diff
@@ -1173,11 +1173,11 @@ declare module "pkcs11js" {
     const CKM_RSA_PKCS_TPM_1_1: number;
     const CKM_RSA_PKCS_OAEP_TPM_1_1: number;
     const CKM_VENDOR_DEFINED: number;
-
+    
```

Comment:
@microshine: I don't know; you could ask at [Definitelytyped/tsd](https://github.com/DefinitelyTyped/tsd). (Or you could always transition to `@types`! See the [blog post](https://blogs.msdn.microsoft.com/typescript/2016/09/22/announcing-typescript-2-0/))